src/openrct2-ui/windows/Ride.cpp

```diff
@@ -97,9 +97,7 @@ enum {
     WIDX_TEST_LIGHT,
     WIDX_OPEN_LIGHT,
     WIDX_RIDE_TYPE,
-    WIDX_RIDE_TYPE_INCREASE,
-    WIDX_RIDE_TYPE_DECREASE,
-    WIDX_RIDE_TYPE_APPLY,
```

Comment:
Removing the Apply button does increase the change of unintended stuff happening is the user misclicks the dropdown. And since changing ride type is quite a hack, the consequences can be very annoying. I would recommend putting it back.

Reply:
Hmm, but wasn't the whole reason for it to be introduced the fact that you didn't have an overview of all ride types at the time? You mentioned non-standard widget types elsewhere in the comments… We don't wait for confirmation for any other dropdowns, either. I personally don't think we should put it back.